packages/core/src/App/app.jsx

```diff
@@ -120,6 +124,20 @@ const AppWithoutTranslation = ({ root_store }) => {
             <Loading />
         );
 
+    React.useEffect(() => {
+        const html = document?.querySelector('html');
```

Comment:
The value of quill tokens is based on css selector: html.light - for light mode, html.dark - for dark mode. At the moment of loading, html tag has no class (for 1 sec), so the value of token is incorrect. Adding class in ui store allows to apply it from the very beginning.